ui/app/components/inference/NewOutput.tsx

```diff
@@ -105,7 +105,7 @@ export function OutputContent({ output, outputSchema }: OutputProps) {
 
 export default function Output({ output, outputSchema }: OutputProps) {
   return (
-    <SnippetLayout className="w-full">
```

Comment:
`w-full` is already applied to this component's root element, so removing should have no impact.